jsapp/js/editorMixins/assetNavigator.es6

```diff
@@ -91,21 +95,24 @@ class AssetNavigatorListView extends React.Component {
         );
     } else {
 
-      window.setTimeout(()=>{
-        this.activateSortable();
-      }, 1);
+      // TODO FIXME
+      // HACK: we activate sortable with timeout so it is rendered :puke:
+      window.setTimeout(() => {this.activateSortable();}, 1);
```

Comment:
I know there's a comment but I'm curious why the timeout is so short and arbitrary :P, does it just need to wait for the list to populate?

Reply:
It probably might be `0`. This is needed because sortable plugin needs the element to exist (i.e. be already rendered by React). Without timeout this line fails: `var $el = $(ReactDOM.findDOMNode(this.refs.liblist));` as it is `null`. We could do it better for sure, probably by implementing some checks that will see if element is already rendered and if sortable was already activated - then we can safely run the function with each render

Reply:
Understood :+1: